inc/blocks/namespace.php

```diff
@@ -191,6 +220,25 @@ function register_post_type() {
 	);
 }
 
+/**
+ * Update the edit post link for XBs to be the parent post.
+ *
+ * @param string $link The original edit post link.
+ * @param int $post_id The post ID.
+ *
+ * @return string The updated edit post link.
+ */
+function update_xb_edit_post_link( string $link, int $post_id ) : string {
+	if ( get_post_type( $post_id ) !== POST_TYPE ) {
+		return $link;
+	}
+
+	$parent_id = wp_get_post_parent_id( $post_id );
```

Comment:
I think this function was just added in WP 5.7, we maybe don't need to worry about back compat in this instance but if this was going to part of a preview release or someone wanted to use it with Altis < 7 it'd break.

Reply:
Docs page says it was added in 3.1. http://developer.wordpress.org/reference/functions/wp_get_post_parent_id/